src/streamlink/plugins/mycujoo_tv.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import useragents
+from streamlink.stream import HLSStream
+
+
+class MycujooTv(Plugin):
+    '''
+    Support for live and archived transmission of the matches and channels on mycujoo.tv
+    '''
+    url_re = re.compile(r'https?://mycujoo.tv/video/[^\?].*[\?&]v?id=(\d+)')
```

Comment:
> VOD - Yes

`https://mycujoo.tv/video/south-bend-fc`

well if you want to support vod's, URLs like this will not work.

they also don't have `twitter:player:stream`